src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs

```diff
@@ -364,7 +364,7 @@ public static void RevokeEverything(PkiOptions pkiOptions)
                 });
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
This test fails for me with:

<pre>
System.AggregateException : One or more errors occurred. (Assert.Equal() Failure
                             ↓ (pos 2)
Expected: [NoError, NoError, RevocationStatusUnknown | OfflineRevocati···
Actual:   [NoError, NoError, Revoked]
</pre>

Are we sure we are disabling the test on that platform for the right reason?

Reply:
I'll go back and look at the exceptions. Also for https://github.com/dotnet/runtime/pull/70821#discussion_r899176346.